src/lib/coda_networking/coda_networking.ml

```diff
@@ -68,8 +68,15 @@ module Rpcs = struct
         let caller_model_of_response = Fn.id
       end
 
-      include T
-      include Register (T)
+      module T' =
```

Comment:
You could make a helper functor for this pattern. Something like `include Register_perf_histogram (struct include M include Master end) (T)` to replace the whole `T'` and `include T' include Register (T')` stuff.